pkg/mapping/processors/assetinventory.go

```diff
@@ -83,7 +84,7 @@ func (p *AssetInventoryProcessor) Process(ctx context.Context, resourceMapping *
 
 	resourceScope, err := parseProject(resourceName)
 	if err != nil {
-		return err
+		return pmaperrors.Wrap(err)
```

Comment:
Don't change this file yet. Not every error here is user facing error. Let's do that in a follow up